src/main/java/com/app/library/service/BorrowService.java

```diff
@@ -70,7 +68,35 @@ public List<BorrowDto> getBorrowsByUser(UUID userId) {
                 .toList();
     }
 
-    private boolean isBookAlreadyBorrowed(UUID bookId) {
+    @Transactional
+    public void returnAllBookForUser(UUID userId) {
+        borrowRepository.findAllByUserId(userId).stream()
+                .filter(e -> BorrowStatus.BORROWED.equals(e.getBorrowStatus()))
+                .forEach(e -> setReturnStatus(e.getId()));
+    }
+
+    private void setReturnStatus(UUID borrowId) {
+        BorrowEntity ReturnStatusBorrow = borrowRepository.findById(borrowId)
+                .orElseThrow();
```

Comment:
What are you throwing here? I don't see any Exception

Reply:
Done